plugins/services/src/js/components/MesosLogView.js

```diff
@@ -336,11 +336,10 @@ class MesosLogView extends mixin(StoreMixin) {
 
     // Show loader since we will start a request for more logs
     return (
-      <div className="pod flush-top">
-        <Loader
-          innerClassName="loader-small"
-          type="ballSpinFadeLoader" />
-      </div>
+      <Loader
+        className="pod pod-tall-bottom flush-right flush-left"
```

Comment:
This loader was actually offset to the right due to some extra margin on the left. I also added `pod-tall-bottom` because the loader didn't appear centered vertically without it, due to some extra padding on the container of this element.